package.json

```diff
@@ -1,32 +1,32 @@
 {
     "name": "universal",
     "description": "Cross platform, core components of the GPII personalization infrastructure.",
-    "version": "0.1.0",
+    "version": "1.0.0",
```

Comment:
Not unless we do some QA and formally verify that this thing works.

But now seems like a pretty good time to do so.